src/main/kotlin/com/example/mykku/member/domain/LikeFeedComment.kt

```
@@ -0,0 +1,20 @@
+@Entity
+class LikeFeedComment(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Long? = null,
+
+    @ManyToOne
+    @JoinColumn(name = "member_id")
+    val member: Member,
+
+    @ManyToOne
+    @JoinColumn(name = "feed_comment_id")
```

detekt (1.23.8)

[warning] The class or object DailyMessage is empty.

(detekt.empty-blocks.EmptyClassBlock)